README.md

```diff
@@ -1,14 +1,19 @@
 # MUSIC MEDIA STRUCTURE
 Think Twitter but for music. Ye, that's what this is.
 
+The deployment is currently broken but the live site can be found here: https://protected-lowlands-14456.herokuapp.com/
+
```

Comment:
@trevorhalvorson  asked for the link to the live site so I included it in the README.